### PR TITLE
120 -convert year filter to slider

### PR DESCRIPTION
**Pull request by Selkubi**
@cforgaci I converted the graduation year filter to a year range slider.

### CODE REVIEW

File: R/app_server.R [hunk 1]
```diff
@@ -7,21 +7,44 @@ app_server <- function(session,input, output) {
 
   ## Get the data  - temporarily added data file in the project. this will mimic the packages final behaviour
 
-  # data_path <- here::here("analysis", "data", "derived_data", "emu_theses_with_text.csv")
-  # emu_theses <- readr::read_csv(data_path)
-  # usethis::use_data(emu_theses, overwrite = TRUE)
-  data("emu_theses")
-
+  data("emu_clean")
```
Comment (Selkubi) [inline R/app_server.R:10]: This is to load the data within a new setup (metadata-text separately)

File: R/app_server.R [hunk 2]
```diff
@@ -7,21 +7,44 @@ app_server <- function(session,input, output) {
...
+                                       unique()|>
+                                       sort() |>
+                                       min(),
+                                     emu_theses$graduation_year |>
+                                       as.character() |>
+                                       unique() |>
+                                       sort() |>
+                                       median())
+                             )
+
```
Comment (cforgaci) [inline R/app_server.R:40]: Can we simplify `min`, `max` and `value` like this (example on `min`)?

```r
min = min(emu_theses$graduation_year)
```

`min()` should work without the previous three lines.

**Review by cforgaci — COMMENTED** [review]
Thanks, @Selkubi, for this! See a comment I made on the arguments of `updateSliderInput()`

File: R/app_server.R (hunk 2 above)
Comment (Selkubi) [reply R/app_server.R:22]: Thanks. I have introduced the suggested changes, please have a look

Comment (cforgaci) [reply R/app_server.R:22]: Looks good, thanks!

**Review by cforgaci — APPROVED** [review]
Thanks for implementing this feature, @Selkubi! We can merge the changes.

**Comment by cforgaci** [comment]
@Selkubi, can you please resolve the conflicts before we merge?

**Pull request merged**